modules/vba_documents/app/sidekiq/vba_documents/upload_remover.rb

```diff
@@ -14,7 +14,7 @@ class UploadRemover
 
     REMOVAL_QUERY = <<-SQL.squish
         status IN ('received', 'processing', 'error', 'success', 'vbms')
-        AND s3_deleted IS NOT True
+        AND s3_deleted IS False
```

Comment:
I tested this query out against my local database, which had four records in "vbms" status, with the following `s3_deleted` values: `true`, `nil`, `false`, `false`. The record with `nil` was created prior to pulling this branch locally. When I tested the query, it returned only two records, the `false` ones. (Note: I temporarily removed the date check from the query to simply my test.) 

Therefore, I think there's an unintended consequence of this change, that for records created before this PR is merged (where the `attribute` default is added), they will not be caught by this job, and therefore not removed from S3.

Reply:
Yes, sorry, I missed your comment about having a separate PR or process for the older records.